native/src/main/java/io/ballerina/stdlib/oauth2/OAuth2Client.java

```diff
@@ -134,44 +131,41 @@ private static SSLContext getSslContext(BMap<BString, ?> secureSocket) throws Ex
         if (cert == null) {
             throw new Exception("Need to configure 'crypto:TrustStore' or 'cert' with client SSL certificates file.");
         }
-        KeyManagerFactory kmf;
-        TrustManagerFactory tmf;
+        KeyManagerFactory kmf = null;
+        TrustManagerFactory tmf = null;
         if (cert instanceof BString) {
             if (key != null) {
+                tmf = getTrustManagerFactory((BString) cert);
                 if (key.containsKey(OAuth2Constants.CERT_FILE)) {
                     BString certFile = key.get(OAuth2Constants.CERT_FILE);
                     BString keyFile = key.get(OAuth2Constants.KEY_FILE);
                     BString keyPassword = getBStringValueIfPresent(key, OAuth2Constants.KEY_PASSWORD);
                     kmf = getKeyManagerFactory(certFile, keyFile, keyPassword);
-                } else {
-                    kmf = getKeyManagerFactory(key);
+                    return buildSslContext(kmf.getKeyManagers(), tmf.getTrustManagers());
                 }
-                tmf = getTrustManagerFactory((BString) cert);
+                kmf = getKeyManagerFactory(key);
                 return buildSslContext(kmf.getKeyManagers(), tmf.getTrustManagers());
-            } else {
-                tmf = getTrustManagerFactory((BString) cert);
-                return buildSslContext(null, tmf.getTrustManagers());
-            }
+            }            
+            tmf = getTrustManagerFactory((BString) cert);
+            return buildSslContext(null, tmf.getTrustManagers());
         }
         if (cert instanceof BMap) {
             BMap<BString, BString> trustStore = (BMap<BString, BString>) cert;
-            if (key != null) {
+            if(key != null){
```

Comment:
Keep these spaces.

I guess if you ran a build these should be identified by the checkstyle plugin. If it doesn't we might have to check what happened there @MohamedSabthar.

Use this [styling guide](https://raw.githubusercontent.com/wso2/code-quality-tools/v1.4/checkstyle/jdk-17/checkstyle.xml) for your IDE.

```suggestion
            if (key != null) {
```

Reply:
Where can I find the report for the failed build? I want to pinpoint the error but can't find the report, do I have to build and run on my local machine for it?

Reply:
Yes, for a detailed report on Checkstyle errors and SpotBugs reports, you should run them locally. Anyway, you should also build and test the project locally before raising the PR. 🙂